orc8r/cloud/helm/orc8r/charts/certs/templates/bootstrapper-certificate.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.create }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
Bootstrapper is just a private key -- no certificate necessary

(That said, if it's easier to have it be a cert instead of just a private key, that's fine. Just add a comment indicating as such.)

Reply:
I have updated the same on `values.yaml` file.